### DISCUSSION TITLE
help me to resolve this error of google collab

### DISCUSSION THREAD

**Discussion by vedansht26** [datasciencecompleteguide/Machine-Learning#61] [General]
OpenCV(4.7.0) /io/opencv/modules/imgproc/src/resize.cpp:4062: error: (-215:Assertion failed) !ssize.empty() in function 'resize'

**Comment by datasciencecompleteguide** [comment]
Hi @vedansht26 
I have already posted the video related to that error
if then also you are facing the issue
Than please share the code with me so that I can figure out where your are stuck
Thanks
